problems/strings.js

```diff
@@ -68,7 +91,12 @@ verbing('go')
  
  */
 
-function verbing() {}
+function verbing(str) { // come back to
```

Comment:
either remove curly braces for one liners, or put on own line (preferred for now). 
Use a code formatter like prettier option + shift + f